sql/migrations/warehouse/000026_drop_wh_upload_id_column.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE wh_schemas DROP COLUMN IF EXISTS wh_upload_id;
```

Comment:
Let's make this nullable in this release. If we revert image, not having this column will create problems. 
In the next release we can drop the column. We can capture a task in linear to not miss this.